src/plugins/lvm-dbus.c

```diff
@@ -2249,7 +2265,7 @@ BDLVMLVdata** bd_lvm_lvs (const gchar *vg_name, GError **error) {
         return NULL;
```

Comment:
Btw. reading the code all over again, there's a number of other leaks in this function - e.g. in this case of return, `matched_lvs` would be leaked. This is endless... would you like me to add a fix for you or would you like to fix that yourself? :-)

Reply:
I certainly haven't looked at all return paths :) This should just be a matter of having ``g_slist_free_full (matched_lvs, g_free)`` before all returns.